modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -39,44 +58,26 @@ $ gadmin backup list
+[console, gsql]
+----
+gadmin backup create <backup_tag> --incremental --base <full_backup_tag>
+----
+This new backup will cover the time range from the creation time of the specified full backup <full_backup_tag> to the current time.
 
-[NOTE]
-====
-because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  we’d better ensure the possible time points are all covered by existing backups. if there is not this kind of backup, you may create a by `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
-====
-
-== Key Benefits
-
-The primary advantage of Point-in-Time Restore is the flexibility it offers, allowing users to restore the database to a specific time point. This feature is particularly useful in scenarios where unwanted changes or data corruption need to be undone without losing other valuable work done after the last backup.
-
-.Key Benefit Scenario
-====
-. A user creates a full backup on __Sunday__.
-
-. During the week, user creates a differential backup on every day 10pm.
-
-. On Thursday, user finds there are unwanted changes made on Wednesday around 2pm.
-
-. User can restore system to 1:55pm of Wednesday instead of the backup made on Tuesday 10pm.
-
-. The changes made between Tuesday 10pm and Wednesday 1:55pm are saved this way.
-====
-
-Summery of key benefits:
-
-* Precise recover time point to save a much as possible correct work made to database.
-
-
-== Limitations and Considerations
-
-Currently, the Point-in-Time Restore feature has these known limitations:
+== Limitations
 
 .Point-in-Time Restore examples.
 image::PIT_restore.png[]
 
 * GSQL Content: The GSQL content, such as schemas and RBAC settings, cannot be restored to the exact version at the provided time point. After a Point-in-Time Restore, these elements will reflect the version of the nearest differential backup used.
 * Precision Window: Point-in-Time Restore operates with a precision window of a few minutes. To ensure optimal data consistency, it is recommended to restore to a time point at least 2-3 minutes before the timestamp of any unwanted changes.
 * Coverage: Point-in-Time Restore cannot be performed for timestamps not covered by an existing differential backup. If the desired time point is not covered, a new differential backup must be created before performing the restore.
-** To maximize the potential of Point-in-Time Restore, users are expected to do backups regulary. A good example can be doing weekly full backup and daily differential backup in days between two full backups.
+
+== Recommendations
+
+To maximize the effectiveness and efficiency of PITR: Perform weekly full backups and daily differential backups. This ensures that the desired restore point is always covered and available when needed.
+
```

Comment:
I'm not happy with this recommendation. It is quite bold ("maximize") without exhibiting much understanding of the tradeoffs. I will send you an alternative idea.  It also does NOT "ensure... always covered".  The user is actually risking losing up to one day's work of data.

Reply:
@victorleeTG 

Temporarily deleted the recommendation